deploy/operator.yaml

```diff
@@ -928,8 +928,8 @@ spec:
           periodSeconds: 10
         resources:
           limits:
-            cpu: 200m
-            memory: 256Mi
+            cpu: 400m
```

Comment:
AFAIK we don't have any specific data, but when Ninjas found the limits to be too low last time, we doubled them, and now a customer ran into the same issue so we are doubling again. It's not unreasonable that we would be running out of memory now, since a lot of additional features have been added to operator since the v0.1.0 release. If we double them now (to 500 mb) we probably won't have to touch them again for a long time. I don't think we have much specific data on this, but I've been under the impression that in other areas of CS it is not unusual to double memory when you run out (e.g. with dynamic arrays). I tried to find resources on memory for containers but wasn't able to find anything.

I do think we should test to see if the CPU limits benefit by being increased or not -- e.g., test 10 installs of a driver with some sidecars with the old cpu limit, then increase it, reinstall the operator, and do the ten installs again and compare the approximate time the csm object takes to go into the ready state with 200m v 400m. If there isn't a significant difference, then I don't think it necessarily makes sense to increase the cpu limit. I documented this suggestion in the Jira defect for this PR as well.

Reply:
@bharathsreekanth AFAIK we don't have any specific data, but when Ninjas found the limits to be too low last time, we doubled them, and now a customer ran into the same issue so we are doubling again. It's not unreasonable that we would be running out of memory now, since a lot of additional features have been added to operator since the v0.1.0 release. If we double them now (to 500 mb) we probably won't have to touch them again for a long time. I don't think we have much specific data on this, but I've been under the impression that in other areas of CS it is not unusual to double memory when you run out (e.g. with dynamic arrays). I tried to find resources on memory for containers but wasn't able to find anything.

I do think we should test to see if the CPU limits benefit by being increased or not -- e.g., test 10 installs of a driver with some sidecars with the old cpu limit, then increase it, reinstall the operator, and do the ten installs again and compare the approximate time the csm object takes to go into the ready state with 200m v 400m. If there isn't a significant difference, then I don't think it necessarily makes sense to increase the cpu limit. I documented this suggestion in the Jira defect for this PR as well.